README.md

```diff
@@ -53,7 +53,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |20| Tab app monetization | This sample shows how to open purchase dialog and trigger purchase flow using teams-js sdk. ||[View][tab-app-monetization#nodejs]|
 |21| Personal Tab with Nav-Bar menu | Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app. ||[View][tab-navbar-menu#ts]|
 |22| App Caching in Meetings | This sample shows how to improve the subsequent loading time of an App. |||[View][app-cache-meetings#nodejs]|
-|23| Personal Tab with Nav-Bar menu | Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app. ||[View][tab-navbar-menu#ts]|
+|23| Tab calendar in Outlook | Interact with the user's calendar, including opening calendar items and composing meetings from personal tab in Outlook.||[View][tab-calendar#ts]|
```

Comment:
This sample shows how to interact with the user's calendar, including opening calendar items and composing meetings from personal tab in Outlook.

Reply:
Fixed